web/src/LoadingEnvironment.jsx

```diff
@@ -27,14 +27,14 @@ import Center from "./Center";
 
 import { EOS_THREE_DOTS_LOADING_ANIMATED as LoadingIcon } from "eos-icons-react";
 
-function LoadingEnvironment() {
+function LoadingEnvironment({ text = "Loading installation environment, please wait." }) {
```

Comment:
NP: maybe we can rename this component to _Loading_

Reply:
Let's do it as part of our changes related to the status API.